docs/supported_apis/dataframe_supported.rst

```diff
@@ -418,7 +418,14 @@ default to pandas.
 +----------------------------+---------------------------+------------------------+----------------------------------------------------+
 | ``to_latex``               | `to_latex`_               | D                      |                                                    |
 +----------------------------+---------------------------+------------------------+----------------------------------------------------+
-| ``to_parquet``             | `to_parquet`_             | D                      |                                                    |
+| ``to_parquet``             | `to_parquet`_             | P                      | **Dask**: Defaults to Pandas implementation as a   |
+|                            |                           |                        | single file. **Ray**: Parallel implementation with |
+|                            |                           |                        | potentially multiple file output only if path      |
+|                            |                           |                        | parameter is a string, does not end with ".gz",    |
+|                            |                           |                        | ".bz2", ".zip", or ".xz" and compression parameter |
+|                            |                           |                        | is not none or "snappy". In these cases, paths     |
+|                            |                           |                        | with Modin on Ray for ``to_parquet`` are           |
+|                            |                           |                        | `directory` and not file paths.                    |
```

Comment:
```suggestion
| ``to_parquet``             | `to_parquet`_             | P                      | **Dask**: Defaults to Pandas implementation and 
|                            |                           |                        | writes a  single output file. 
|                            |                           |                        | **Ray**: Parallel implementation only if path  |
|                            |                           |                        | parameter is a string; does not end with ".gz",    |
|                            |                           |                        | ".bz2", ".zip", or ".xz"; and compression parameter |
|                            |                           |                        | is not ``None`` or "snappy". In these cases, the |
|                            |                           |                        | ``path`` parameter specifies a directory where |
|                            |                           |                        | one file is written per row partition of the Modin dataframe. |
```

I think this wording is a little clearer. I have probably messed up the table formatting a little, or at least made some of the lines too long, so if you accept this suggestion, you might have to clean that up.